R/openapi-types.R

```diff
@@ -70,6 +70,24 @@ add_api_info_onLoad <- function() {
     as.character,
     location = c("query", "path")
   )
+  addApiInfo(
+    "date-time",
+    c("POSIXct", "POSIXt"),
+    "[^/]+",
+    lubridate::as_datetime,
```

Comment:
Is there a `{base}` method we can use? `{lubridate}` is currently not a direct dependency.

--------------------

If keeping `{lubridate}`... we'll need to wrap the method as to not cache the `{lubridate}` function at `{plumber}`'s installation time. 

Ex:
```r
function(x) { lubridate::as_datetime(x) },
```

Reply:
I don't know.. 

I trust `lubridate` more than base. I feel like I have had issues where base methods haven't done as well with timezone issues... but I don't have specific examples, and I fully admit that I'm not a long-term R coder, and I know way more tidyverse than base R.

Reply:
Resolved - lubridate in as Suggests